README.md

```diff
@@ -204,7 +206,7 @@ waterwheel.populateResources()
 ```
 
 `.setField()` accepts 3 arguments
-  - `nid`: The id of the entity you are setting field values on
+  - `token`: The id of the entity you are setting field values on
```

Comment:
Why use "token" here but "identifier" in .patch()? Seems "identifier" would be better in both places.